tests/pcap-log-uncompressed-01/test.yaml

```diff
@@ -0,0 +1,12 @@
+requires:
+  min-version: 8
+      
+pcap: ../bug-2482-01/proxyCONNECT_443.pcap
+
+args:
+  - --runmode=single
+
+checks:
+    - shell:
+        args: find . -type f -name 'log.pcap.1523389*.*' | wc -l | xargs
```

Comment:
What does `1523389` mean here ?

Would `file-compare` be a good check here rather than resorting to shell ?

Reply:
Is there not a config format string that you can use to ignore the timestamp ?

Reply:
I explicitly added it to test it

Reply:
which exposed https://redmine.openinfosecfoundation.org/issues/7034

Reply:
Oh... 👏